source/OutfitInfoDisplay.cpp

```diff
@@ -561,6 +566,8 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 			{
 				attributeLabels.emplace_back(VALUE_NAMES[i].first + PER_SECOND);
 				attributeValues.emplace_back(Format::Number(60. * values[i] / reload) + VALUE_NAMES[i].second);
+				if(fullDropoff != 1 && VALUE_NAMES[i].first.find("damage") != string::npos)
+					attributeValues.back() += " - " + Format::Number(60. * fullDropoff * values[i] / reload) + VALUE_NAMES[i].second;
```

Comment:
```suggestion
					attributeValues.emplace_back(" - " + Format::Number(60. * fullDropoff * values[i] / reload) + VALUE_NAMES[i].second);
```
this is what I meant with consistency, should have made myself clear

Reply:
That seems to break everything. 
![image](https://github.com/endless-sky/endless-sky/assets/18634983/3e036b1d-9c14-47d4-8c9f-1da0f481b731)

Reply:
yeah you're adding another element if you do that @RisingLeaf which means a new line

Reply:
ah, in that case dont do it